### PR TITLE
MBM-147/148 Myrial effigy and suit recipes (done by Michael)

### PR DESCRIPTION
**Pull request by MattKirchh**
GitHub was stupid again and merged 2 PR's when I only merged 1. GitHub claims these changes have been committed to `develop` and thus a PR can't be made from that branch, so I just copied the changes to a new branch here.

### CODE REVIEW

File: src/main/resources/data/minebound/recipes/myrial_suit_boots_from_armor_forge.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  "armor": "minebound:myrial_effigy_boots",
+  "ingredients": [
+    {
+      "item": "minebound:illuminant_steel_boots"
+    },
+    {
+      "item": "minecraft:ghast_tear"
+    },
+    {
+      "item": "minecraft:glowstone_dust"
```
Comment (insertStringHere) [inline src/main/resources/data/minebound/recipes/myrial_suit_boots_from_armor_forge.json:12]: Should be gold ingot

Comment (insertStringHere) [reply src/main/resources/data/minebound/recipes/myrial_suit_boots_from_armor_forge.json:12]: @solidwaterslayer

**Comment by MattKirchh** [comment]
The recipes that require a Myrial armor piece in the center slot do not actually check for what's in that center slot. Only what's in the surrounding 5 slots.

**Review by YonatanWN — APPROVED** [review]

**Comment by insertStringHere** [comment]
> The recipes that require a Myrial armor piece in the center slot do not actually check for what's in that center slot. Only what's in the surrounding 5 slots. That's not intentional, is it?

fixed

**Review by griffinsellinger — APPROVED** [review]
built and tested - worked fine

**Pull request merged**